spyderlib/plugins/workingdirectory.py

```diff
@@ -247,6 +248,8 @@ def register_plugin(self):
         """Register plugin in Spyder's main window"""
         self.redirect_stdio.connect(self.main.redirect_internalshell_stdio)
         self.main.console.shell.refresh.connect(self.refresh_plugin)
+        iconsize = 24 
+        self.setIconSize(QSize(iconsize, iconsize))

```

Comment:
Just a note for a future PR: could we use something like this `iconsize` value to adjust the size of our toolbars?